client.go

```diff
@@ -51,7 +50,7 @@ const (
 
 // Connect to ovn, using endpoint in format ovsdb Connection Methods
 // If address is empty, use default address for specified protocol
-func Connect(endpoints string, tlsConfig *tls.Config) (*OvsdbClient, error) {
+func Connect(endpoints string, database string, tlsConfig *tls.Config) (*OvsdbClient, error) {
```

Comment:
Having only one database, should we allow the client not to specify the database? maybe an optional (default?) behavior could be to run ListDbs() to get the available databases and use the first one (and warn if more than one). So the API could be as simple as:
`c, err := Connect("unix:/var/run/ovn/ovn_nb.sock")`
wdyt?

Reply:
I don't think so. The behaviour would be too unpredictable.

For example, latest versions of OVS have both the `Open_vSwitch` and `_Server` schemas in the ovsdb-server.
I'm not sure if there are guarantees re: ordering of what comes back from `ListDbs`, but I guess it's possible that you could get the `_Server` schema as the default, which probably isn't what you wanted.

Reply:
Right! in that case the current API is perfect. Thanks.